core/dbt/task/run.py

```diff
@@ -125,25 +125,23 @@ def run_hooks(self, adapter, hook_type: RunHookType, extra_context):
 
             hook_text = '{}.{}.{}'.format(hook.package_name, hook_type,
                                           hook.index)
-            print_hook_start_line(hook_text, idx, num_hooks)
-
             hook_meta_ctx = HookMetadata(hook, self.index_offset(idx))
             uid_ctx = UniqueID(hook.unique_id)
-            with uid_ctx, hook_meta_ctx, startctx:
-                logger.debug('on-run-hook starting')
+            running_ctx = DbtModelState({'node_status': 'running'})
```

Comment:
Would it make more sense to put the entire rest of this method under a `with uid_ctx:` and tab everything in?

Reply:
yep, that's a really good idea!